### PR TITLE
add firefox to scenario.py and other tweaks

### PR DESCRIPTION
**Pull request by lalo**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class MultimodalWebSurferAgent(ConversableAgent):
 
     DEFAULT_DESCRIPTION = "A helpful assistant with access to a web browser. Ask them to perform web searches, open pages, and interact with content (e.g., clicking links, scrolling the viewport, etc., filling in form fields, etc.)"
 
-    DEFAULT_START_PAGE = "https://www.bing.com/"
+    DEFAULT_START_PAGE = "https://www.berkshirehathaway.com/"
```
Comment (afourney) [inline autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py:54]: This seems like you left in some testing code? Not a huge deal, but I wonder if you could switch it back to bing for a more general audience?

File: autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py [hunk 2]
```diff
@@ -117,12 +117,22 @@ def __init__(
 
         # Create the context -- are we launching a persistent instance?
         if chromium_data_dir is None:
-            browser = self._playwright.chromium.launch(**launch_args)
+            if chromium_channel == "chromium":
+                browser = self._playwright.chromium.launch(**launch_args)
+            elif chromium_channel == "firefox":
```
Comment (afourney) [inline autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py:122]: I wonder if we should rename this variable if applied to Firefox. It is no longer just the chromium channel.

File: autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py [hunk 3]
```diff
@@ -117,12 +117,22 @@ def __init__(
...
+                )
             self._context = browser.new_context(
                 user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.0"
             )
         else:
-            self._context = self._playwright.chromium.launch_persistent_context(chromium_data_dir, **launch_args)
+            if chromium_channel == "chromium":
+                self._context = self._playwright.chromium.launch_persistent_context(chromium_data_dir, **launch_args)
+            elif chromium_channel == "firefox":
+                self._context = self._playwright.firefox.launch_persistent_context(chromium_data_dir, **launch_args)
```
Comment (afourney) [inline autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py:135]: Likewise, here... it's no longer a chromium_data_dir, but a browser_data_dir

File: autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py (hunk 2 above)
Comment (lalo) [reply autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py:122]: I wasn't sure of how many other variations of 'scenario.py' exist (and maybe in other prs) so for now preferred not to break the arg. I can check this out though.

**Review by afourney — APPROVED** [review]

**Pull request merged**